### PR TITLE
Standardize view IDs per upcoming extenders documentation

### PR DESCRIPTION
**Pull request by lauren-li**
## Proposed changes

This PR standardizes the formatting of Zowe Explorer's view IDs. The formatting follows the extender documentation updated in PR #1290.

- `zowe.explorer` is changed to `zowe.ds.explorer`
- `zowe.jobs` is changed to `zowe.jobs.explorer`
- `zowe.uss.explorer` remains the same
- This PR should not have any direct effects on Zowe Explorer's UI/UX. However, any extenders that currently hook into Zowe Explorer's context menus using the view IDs `zowe.explorer` or `zowe.jobs` will need to update those ID references to `zowe.ds.explorer` and `zowe.jobs.explorer`, respectively.

## Release Notes

Milestone: TBD. Dependent on conformance criteria approval by the Onboarding squad.

Changelog: Standardize the formatting of Zowe Explorer's view IDs.

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [x] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [x] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

### CODE REVIEW

File: packages/zowe-explorer/__tests__/__theia__/theia/Locators.ts
Comment (lauren-li) [inline packages/zowe-explorer/__tests__/__theia__/theia/Locators.ts:56]: @deepali-hub @crawr Should line 56 be as follows?:
```
    jobsPanelId: "plugin-view:zowe.jobs.explorer",
```

**Comment by VitGottwald** [comment]
Thank you @lauren-li ! I asked a few teams internally to verify this will not break their extensions. I am currently waiting for their response. Since it is summer and many people have taken vacation it might take a bit longer than usual.

**Review by jellypuno — DISMISSED** [review]
lgtm!

**Comment by VitGottwald** [comment]
I am currently aware of two extension on the Broadcom side that will have to be updated with this change. Waiting for one more to respond.

**Comment by VitGottwald** [comment]
Just heard back from the last team, and they are unaffected. So we have two extensions to update after this is released.

**Review by VitGottwald — APPROVED** [review]

**Comment by lauren-li** [comment]
Thanks @VitGottwald for getting extender feedback on these changes! I believe HCL also has an extension that would be affected by this PR.

Per @jellypuno, this PR should not be merged in until the v1 conformance criteria is approved by the Onboarding squad and we are able to disperse information allowing extenders time to update their extensions.

**Comment by jellypuno** [comment]
@phaumer @lauren-li Were you able to communicate with the HCL team?

**Comment by lauren-li** [comment]
@jellypuno We will need to wait for @phaumer to return in order to communicate with the HCL team (unless one of them reaches out to us first). However, I was able to verify through installation that the extension [HCL z Tools](https://marketplace.visualstudio.com/items?itemName=hcl.hcl-ztools) would be affected by the changes in this PR.

**Comment by jellypuno** [comment]
Ok we will wait. Worst case scenario, we are going to announce this PR in the ZE channel. Hopefully they will be informed through that.

**Comment by lauren-li** [comment]
@jellypuno @VitGottwald Per the Zowe Explorer squad's decision, I have changed the base branch for this PR to the `next` branch. Could you please take another look at this PR? Thanks in advance!

**Comment by lauren-li** [comment]
The `tar` vulnerabilities in this PR seem to be shared with the base `next` branch, and should be resolved once the `next` branch is updated with the latest `master` branch.

**Review by jellypuno — DISMISSED** [review]

**Review by phaumer — CHANGES_REQUESTED** [review]
I see conflicts plus this branch does not build. It seems to point to an invalid directory "src/src":

```txt
[07:44:57] 'cleanTask' errored after 33 ms
[07:44:57] Error: EINVAL: invalid argument, rmdir '/Users/phaumer/Dev/Zowe/vscode-extension-for-zowe/packages/zowe-explorer/out/src/src/utils'
```

**Review by VitGottwald — DISMISSED** [review]
LGTM

**Comment by lauren-li** [comment]
> I see conflicts plus this branch does not build. It seems to point to an invalid directory "src/src":
> 
> ```
> [07:44:57] 'cleanTask' errored after 33 ms
> [07:44:57] Error: EINVAL: invalid argument, rmdir '/Users/phaumer/Dev/Zowe/vscode-extension-for-zowe/packages/zowe-explorer/out/src/src/utils'
> ```

@phaumer Apologies about the conflicts; those have been resolved now! This PR does not make any changes to any config files such as webpack, tsconfig, or gulp, so I'm not quite sure why there is a new build error for you. However, the error references`.../packages/zowe-explorer/out/src/src`, which corresponds to the folder we generate to contain JSON metadata for localization. (Usually, this folder gets generated during the build.)

The commands I used to build and test were `yarn && yarn workspace vscode-extension-for-zowe vscode:prepublish`*. Do those work for you? What commands did you use?
(*Running `yarn` seems to include `yarn build`.)

**Review by phaumer — APPROVED** [review]

**Review by jellypuno — APPROVED** [review]

**Pull request merged**
